FantasmoSDK/app/src/main/java/com/fantasmo/sdk/models/analytics/TotalDeviceRotationAccumulator.kt

```diff
@@ -18,15 +18,18 @@ class TotalDeviceRotationAccumulator {
     // Being Min=[0], Max=[1] and Spread=[2]
     // Roll is rotation on X axis
     // Max and Min valid range is [−π,π], Roll spread ∈ [0,2π]
-    var roll = floatArrayOf(0f, 0f, 0f)
+    var roll = floatArrayOf(Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY, 0f)
+        private set
 
     // Yaw is rotation on Y axis
     // Max and Min valid range is [−π,π], Yaw spread ∈ [0,2π]
-    var yaw = floatArrayOf(0f, 0f, 0f)
+    var yaw = floatArrayOf(Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY, 0f)
+        private set
 
     // Pitch is rotation on Z axis
     // Valid range is [−π/2,π/2], Pitch spread ∈ [0,π]
-    var pitch = floatArrayOf(0f, 0f, 0f)
+    var pitch = floatArrayOf(Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY, 0f)
```

Comment:
hmm, shouldn't these begin as 0, since they represent the start degrees of the phone? Also I'm pretty sure we assume (just a haphazard guess really) that the phone begins with a -90, +90 Z axis (i.e. someone is holding the phone naturally, with the front camera facing towards them, in an upright position)